CMakeLists.txt

```diff
@@ -142,40 +150,61 @@ if(WIN32)
 endif()
 
 # Find or build hypre if the option is set to use it
-if(USE_HYPRE)
+if(USE_HYPRE OR USE_HYPRE_NVIDIA OR USE_HYPRE_AMDGPU OR USE_HYPRE_INTELGPU)
     # Unless forced to use the system version, download and build hypre
     if(NOT(USE_SYSTEM_HYPRE))
         include(FetchContent)
         # As we are not using the system hypre, we need to choose the version we
         # want
-        set(HYPRE_GIT_VERSION "2.32.0" )
+        set(HYPRE_GIT_VERSION "v2.32.0-24-63331f19c" )
         FetchContent_Declare(
             HYPRE
             GIT_REPOSITORY https://github.com/hypre-space/hypre.git
-            # Currently we need to refer directly to a commit which contains a
-            # patch to handle HYPRE_FMANGLE
-            GIT_TAG        6d01417697aa84d118eb1e1eb91315df42575be5 # v${HYPRE_GIT_VERSION}
+            GIT_TAG        63331f19c72b2ec363824907c0575a6edf7bbf3c # From: 01/27/25 # v${HYPRE_GIT_VERSION}
```

Comment:
@rmcdermo this is the minimum required version of hypre for this to work. This commit hash is located at hypre's master branch and will be added in the upcoming release

Reply:
@victorapm OK, understood.  Thanks!  We can add to FDS master branch once there is a new hypre release.